api/src/services/__tests__/competenciesService.ts

```diff
@@ -62,4 +64,28 @@ describe('competenciesService', () => {
       });
     });
   });
+
+  describe('updateCompetency', () => {
+    it('should update the specified competency if the current user is the one who originally authored the competency', async () => {
+      const id = 2;
+      const label = 'label';
+      const description = 'description';
+      mockQuery(
+        'update `competencies` set `label` = ?, `description` = ? where `id` = ?',
+        [label, description, id],
+        1
+      );
+      await updateCompetency(id, label, description);
+    });
+    it('should authorizeCompetencyUpdate for a user who authored the competency', async () => {
```

Comment:
This example is testing the `authorizeCompetencyUpdate` function, but it's inside of the `describe('updateCompetency'` block. It looks like a new block should be created for testing the function.

Also, using the function name in the test description makes it hard to understand what the test is meant to assert. I think the test description should be updated to explain what's being asserted in the test itself.

Reply:
I will separate the test block for the authorization of the competency update.  I will make changes to the test description as suggested.